stac-example/src/main/scala/geotrellis/stac/api/SearchFilters.scala

```diff
@@ -55,14 +57,28 @@ object SearchFilters {
   implicit val geometryIntersectionSemigroup: Semigroup[Geometry] =
     Semigroup.instance { (left, right) => left.intersection(right) }
 
+  implicit val bboxIntersectionSemigroup: Semigroup[Bbox] =
+    Semigroup.instance { (left, right) =>
```

Comment:
Is this `Semigroup.instance` a shortcut to avoid using the anonymous class syntax?

Reply:
Exactly!

Reply:
Cool, gonna use that over in the other semigroup PR if I can. 

The more of this functional stuff I do, the more I think Scala needs more ways to do the same thing.... _/s_